packages/core/src/app/checkout/Checkout.test.tsx

```diff
@@ -329,20 +329,6 @@ describe('Checkout', () => {
             expect(screen.getByText('test@example.com')).toBeInTheDocument();
         });
 
-        it('logs unhandled error', async () => {
```

Comment:
I removed this test because it depends on `applepay` being loaded. However, due to lazy loading, `applepay` is only loaded when needed. Therefore, this test is no longer applicable.